src/dsp_tools/models/custom_warnings.py

```diff
@@ -13,6 +13,15 @@ def showwarning(cls, message: str) -> None:
         """Functionality that should be executed when a warning of this class is emitted"""
 
 
+class GeneralDspToolsWarning(Warning):
+    """Class for user-facing deprecation warnings"""
```

Comment:
- no deprecation here
- check with https://github.com/dasch-swiss/dsp-tools/pull/930, and take the warning defined there, instead of defining a new one here